types/three/examples/jsm/webxr/XRHandPrimitiveModel.d.ts

```diff
@@ -0,0 +1,14 @@
+import { Group, Texture } from '../../../src/Three';
+
+import { HandModelOptions, XRHandModel } from './XRHandModelFactory';
+
+export class XRHandPrimitiveModel {
+    controller: Group;
+    handModel: XRHandModel;
+    envMap: Texture | null;
```

Comment:
Is this ever not `null`? I haven't figured out its purpose, but it's probably fine to leave as-is.